packages/expo-updates/src/Updates.types.ts

```diff
@@ -84,6 +84,54 @@ export type UpdateEvent = {
   message?: string;
 };
 
+/**
+ * An object representing a single log entry from expo-updates logging on the client.
+ */
+export type UpdatesLogEntry = {
+  /**
+   * The time the log was written, in seconds since Jan 1 1970 UTC.
+   */
+  timestamp: number;
+  /**
+   * The log entry message.
+   */
+  message: string;
+  /**
+   * One of the defined code values for expo-updates log entries.
+   */
+  code: UpdatesLogEntryCode;
+  /**
+   * The level or severity of the entry (fatal, error, warn, info, debug, trace)
+   */
+  level: string;
```

Comment:
something like
```suggestion
  level: 'fatal' | 'error' | 'warn' | 'info' | 'debug' | 'trace';
```
would be more specific

Reply:
I'll create an enum for this

Reply:
Done